external-crates/move/move-compiler/tests/move_check/typing/return_any_type.exp

```diff
@@ -14,3 +26,15 @@ warning[W09005]: dead or unreachable code
   │
   = This warning can be suppressed with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
 
+warning[W09004]: unnecessary trailing semicolon
+  ┌─ tests/move_check/typing/return_any_type.move:9:23
+  │
+9 │         foo(return ());
```

Comment:
Ditto here, I don't think we should be emitting this here, or shoudl at least give a different error